.evergreen/config.yml

```diff
@@ -137,14 +137,16 @@ buildvariants:
     tasks:
       - run-prebuild
       - run-prebuild-force-publish
+    expansions:
```

Comment:
Did you intentionally bump the node version to 18 for macos?  Previously it defaulted to Node14

Reply:
Yes that was intentional.

Reply:
Per our discussion I've made it 14 again for consistency with the other distros. Even though it was "hydrogen" previously.